capa/ida/plugin/form.py

```diff
@@ -227,7 +227,7 @@ def get_values(self):
 class CapaExplorerForm(idaapi.PluginForm):
     """form element for plugin interface"""
 
-    def __init__(self, name):
+    def __init__(self, name, option=0):
```

Comment:
i'd recommend:
  - a boolean for this, rather than integer
  - a human readable name

Reply:
`run` receives a number: https://www.hex-rays.com/blog/scriptable-plugins

I can change the name by `analyze` or something else. But I was thinking we may want to add other options. For example: https://github.com/fireeye/capa/issues/547

🤔

Reply:
ah, ok, thanks for the explanation. maybe a tiny comment to this effect, then?

if we're going to support multiple options, but only a single arg, maybe we could use a bitmask here?

Reply:
yes and use constants a enum?

Reply:
Yes! bitmask + enum is what I was thinking too when/if there are more options. Should we use it for one as well?

I am not sure this is the best alternative (it is probably the easiest/fastest). Ideally the explorer would run the analysis for both views when opening the explorer always. Even for https://github.com/fireeye/capa/issues/547, I would prefer `select all` to be the default. If we disagree on the defaults, that would be a reason to have options.

Reply:
Yes! bitmask + enum is what I was thinking too when/if there are more options. Should we use it for one as well?

Reply:
I think so